qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -128,6 +136,12 @@ def change_virtio_media(cdrom_virtio):
         static_ip_address = utils_net.get_guest_ip_addr(session_serial, virtio_nic_mac, os_type='windows')
         if static_ip_address != params["static_ip"]:
             test.fail("Static ip is lost after upgrade driver,current ip is %s" % static_ip_address)
+        static_dns_address = utils_net.get_windows_nic_attribute(session_serial, global_switch="nicconfig",
+                                                                 key="MACAddress", value=f"{virtio_nic_mac}",
+                                                                 target="DNSServerSearchOrder")
+        static_dns_address = static_dns_address.strip('{}').strip('"')
+        if static_dns_address != params["static_dns"]:
+            test.fail("Static dns is lost after upgrade driver,current dns is %s" % static_dns_address)
```

Comment:
@heywji Shall we consider package these lines into a function as you repeat exactly the same lines twice.

Reply:
OK! I will change it now.

Reply:
Hi @vivianQizhu, I have put these lines into a function. Could you please check them again?

Test Result: Pass
```
Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_update.from_old_installer.q35: PASS (955.86 s)
```